### DISCUSSION TITLE
CPU/Memory/Disk Space Requirements for Hosting Kimai for an Entity with 400 employees

### DISCUSSION THREAD

**Discussion by sch-moozer** [kimai/kimai#4709] [General]
Hi Kevin,

We are looking to deploy Kimai for a non-profit organization with 400 employees. We estimate that no more than a hundred employees will access the server simultaneously.

Do you have any recommendations for CPU/Memory/disk space for hosting Kimai? 
Do you have any thoughts on hosting the database on a separate server? We have been considering this option to reduce the possibility of running into performance issues down the line.

**Comment by kevinpapst** [comment]
400 users is nothing... 
Kimai get a few pagehits per user per day. 
That will run on a Banana Pi or similar micro computer.

Want to use a "serious server"? 
Get a 12€ Hetzner Cloud machine with 8GB RAM: https://www.hetzner.com/cloud/
That's more than enough.

**Comment by eboujobe** [comment]
Thank you for getting back to us. We are considering an in-house mysql database as well. Can we host the mysql database on the Hetzner server or do you recommend using an external database like Amazon RDS which handles data backup, load balancers etc if there is a need.

Thank you for the great open-source software you developed for the community.
